ast/ddl.go

```diff
@@ -1622,6 +1624,13 @@ func (n *TableOption) Restore(ctx *RestoreCtx) error {
 	case TableOptionStorageMedia:
 		ctx.WriteKeyWord("STORAGE ")
 		ctx.WriteKeyWord(n.StrValue)
+	case TableOptionStatsSamplePages:
+		ctx.WriteKeyWord("STATS_SAMPLE_PAGES ")
+		ctx.WritePlainf("= %d", n.UintValue)
```

Comment:
If `n.UintValue == 0` the restored output should be `STATS_SAMPLE_PAGES = DEFAULT`